src/js/components/Menu/Menu.js

```diff
@@ -351,11 +357,12 @@ const Menu = forwardRef((props, ref) => {
 
                   return (
                     // eslint-disable-next-line react/no-array-index-key
-                    <Box key={index} flex={false}>
+                    <Box key={index} flex={false} role="none">
```

Comment:
Do we need role="none" here?

Reply:
This is because `Box` is the parent of each menuitem since a `menuitem` is not directly below a `menu` the accessibility tree might display a bit differently but I don't think it will change how it's announced.

Reply:
Thanks for the explanation!